.idea/encodings.xml

```diff
@@ -4,30 +4,20 @@
     <file url="file://$PROJECT_DIR$" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/catalog" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/catalog/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/catalog/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/control" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/control/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/control/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/currency" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/currency/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/currency/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/entitlement" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/entitlement/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/entitlement/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/invoice" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/invoice/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/invoice/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/notification" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/notification/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/notification/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/payment" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/payment/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/payment/src/main/resources" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/usage" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/usage/src/main/java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/usage/src/main/resources" charset="UTF-8" />
```

Comment:
Are you on the latest version of IntelliJ? I had just fixed this on 5e1c8d53c7fb85a89e89c1512f4d49acce29e0cf.

Reply:
<img width="461" alt="Screen Shot 2022-03-31 at 3 35 08 PM" src="https://user-images.githubusercontent.com/246910/161160885-7cb670b3-a856-4d67-8d65-65947d07242d.png">